contents/blog/best-amplitude-alternatives.mdx

```diff
@@ -110,11 +108,11 @@ Mixpanel and Amplitude are the original product analytics platforms and have bat
 <ComparisonTable column1="Mixpanel" column2="Amplitude">
   <ComparisonRow column1={true} column2={true} feature="Self-serve" description="Free to try, no mandatory sales calls" />
   <ComparisonRow column1={true} column2={true} feature="Product analytics" description="Filter and visualize data with trends, funnels, paths, and retention analysis" />
-  <ComparisonRow column1={false} column2={false} feature="Autocapture" description="Capture events without manual instrumentation" />
+  <ComparisonRow column1={false} column2={true} feature="Autocapture" description="Capture events without manual instrumentation" />
   <ComparisonRow column1={true} column2={true} feature="Group analytics" description="Track metrics at the account or company level" />
   <ComparisonRow column1={"Manual"} column2={true} feature="A/B testing" description="Run multivariate tests and see the impact of changes with custom goals and reports" />
   <ComparisonRow column1={false} column2={true} feature="Feature flags" description="Deploy features safely with targeting, percentage rollouts, and instant rollbacks" />
-  <ComparisonRow column1={false} column2={false} feature="Session replay" description="Watch real users on your site; discover friction points" />
+  <ComparisonRow column1={true} column2={true} feature="Session replay" description="Watch real users on your site; discover friction points" />
```

Comment:
Do we mention that session replay is an enterprise only feature for Mixpanel?

![CleanShot 2024-10-07 at 09 35 46](https://github.com/user-attachments/assets/7444e25f-79d9-4e13-80af-eedf5bc15a70)

Reply:
We don't. Can add it into the copy, but I'll leave the table as it's easier to avoid too much caveated in the tables, otherwise we have to do the same for things in the Team plans etc.